src/DynamoCore/Models/NodeModel.cs

```diff
@@ -1092,6 +1094,20 @@ internal double GetPortVerticalOffset(PortModel portModel)
         /// </summary>
         public void RegisterInputPorts()
```

Comment:
We mark code obsolete to indicate that we will eventually remove it. Not that we're necessarily removing it right now.